src/sentry/conf/server.py

```diff
@@ -1531,7 +1531,7 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
         "PARSER_WHITELIST": ["rest_framework.parsers.JSONParser"],
         "POSTPROCESSING_HOOKS": [
             "sentry.apidocs.hooks.custom_postprocessing_hook",
-            "drf_spectacular.hooks.postprocess_schema_enum_id_removal",
+            "drf_spectacular.hooks.postprocess_schema_enums",
```

Comment:
This processing is necessary for the new endpoint to remove the `null` values from the enum being added, but also adds a lot more documentation to existing enum fields. The diff is too large to show all the changes though